lib/task-data/tasks/boot-livecd.js

```diff
@@ -0,0 +1,22 @@
+// Copyright 2015, EMC, Inc.
+
+'use strict';
+
+module.exports = {
+    friendlyName: 'Boot LiveCD',
+    injectableName: 'Task.Os.Boot.LiveCD',
+    implementsTask: 'Task.Base.Os.Install',
+    options: {
+        profile: 'boot-livecd.ipxe',
+        completionUri: 'renasar-ansible.pub',
+        version: 'livecd',
```

Comment:
The ISO image is user generated and can be any platform/version.  As there are no fixed/expected names to start with, 'livecd' seems like a good guess at a name that someone might use when creating their own livecd.